libtiledbvcf/src/read/reader.cc

```diff
@@ -959,6 +1030,17 @@ void Reader::prepare_variant_stats() {
   af_filter_->compute_af();
 }
 
+void Reader::prepare_allele_count() {
+  init_for_allele_count();
```

Comment:
Do we need `Reader::init_for_allele_count` and `Reader::init_allele_count_reader_for_export`? If not, it would be simpler to create the `ac_reader_` object right before using it below.

Reply:
This is broken up to separate the Embedded query from the TileDB-VCF read, as it's necessary to provide buffer sizes prior to the latter.